doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
+
+The **inventory sheet** refines an inventory plan by excluding formats by adding/removing sets of content items.
+
+**Audit Sheet**: is what a the department is going to audit. It is the inventory plan refined twice.
+
+## Audit plan: input of an audit
+
+The data input for an Audit is modelled in an **audit plan**, not in a `theme`. An audit plan is a set of content items conformed this way:
+
+- An inventory plan (set of rules that define a theme)
+- A set of formats to exclude
+- A set of content items to include
```

Comment:
Does this mean we can include content from other Themes in an audit? Is that something we'd ever want to do? And if so, what meaning does a "Theme" actually have?